static/js/components/PersonalTab.js

```diff
@@ -23,21 +33,76 @@ class PersonalTab extends React.Component {
     ui:             UIState,
     nextStep:       () => void,
     prevStep:       () => void,
+    dashboard:      DashboardState,
+    setProgram: Function,
+    addProgramEnrollment: Function,
   };
 
+  programListing = (programs: Array<Program>) => {
+    const makeMenuItems = R.map(program => (
+      <MenuItem value={program.id} key={program.id} primaryText={program.title} />
+    ));
+    const sortPrograms = R.sortBy(R.compose(R.toLower, R.prop('title')));
+    const menuItems = R.compose(makeMenuItems, sortPrograms);
+    return menuItems(programs) ;
+  }
+
+  setProgramHelper = (event: Event, key: string, value: string) => {
+    const {
+      dashboard: { programs },
+      setProgram,
+    } = this.props;
+    let selected = programs.find(program => program.id === value);
+    let enrollment: ProgramEnrollment = {
+      id: selected.id,
+      title: selected.title
+    };
+    setProgram(enrollment);
+    this.setState({program: selected});
+  }
+
+  selectProgram = () => {
+    const { dashboard: { programs } } = this.props;
+
+    return (
+      <SelectField
+        value={this.state.program ? this.state.program.id : null}
+        style={{width: "65%"}}
+        hintText="Select Program"
+        onChange={this.setProgramHelper}
+        className="program-select"
+      >
+        { this.programListing(programs) }
+      </SelectField>
+    );
+  }
+
   render() {
     return (
       <div>
+        <Card shadow={1} className="program-select">
+          <div className="section-header">Which MicroMasters program are you signing up for?</div>
+          <br/>
+          { this.selectProgram() }
+        </Card>
         <Card shadow={1} className="profile-form">
           <PersonalForm {...this.props} validator={personalValidation} />
           <ValidationAlert {...this.props} />
         </Card>
-        <ProfileProgressControls
-          {...this.props}
-          nextBtnLabel="Next"
-          isLastTab={false}
-          validator={personalValidation}
-        />
+        <div className="terms-of-service-container">
```

Comment:
@amir-qayyum-khan we're moving this message back to the sign-up modal on the home & program pages. 

Please remove it from this form.